ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/port-channel-interfaces.j2

```diff
@@ -352,6 +352,14 @@ interface {{ port_channel_interface.name }}
 {%     elif port_channel_interface.bfd.echo is arista.avd.defined(false) %}
    no bfd echo
 {%     endif %}
+{%     if port_channel_interface.bfd.neighbor is arista.avd.defined %}
+   bfd neighbor {{ port_channel_interface.bfd.neighbor }}
+{%     endif %}
+{%     if port_channel_interface.bfd.per_link.rfc_7130 is arista.avd.defined(true) %}
+   bfd per-link rfc-7130
+{%     elif port_channel_interface.bfd.per_link.rfc_7130 is arista.avd.defined(false) %}
```

Comment:
We should not check on false. I think we need an `enabled` key instead. Then configure `bfd per-link rfc_7130` if enabled *and* rfc_7130 is true and `bfd per-link.
Implement with a outer `if` checking for enabled and then if/else on 7130.